src/main/java/odd/jobs/controllers/UserCrudOperationsController.java

```diff
@@ -17,8 +22,35 @@ public UserCrudOperationsController(UserCrudOperationsService userCrudOperations
         this.userCrudOperationsService = userCrudOperationsService;
     }
 
+    @GetMapping("/users")
+    public List<User> getAllUsers()
+    {
+        return userCrudOperationsService.getAllUsers();
+    }
+
+
+    @GetMapping("/users/{id}")
+    public UserDetails getUserById(@PathVariable long id) throws NotFoundException {
+        return userCrudOperationsService.getUserById(id);
+    }
+    /*
     @GetMapping("/users/{name}")
     public UserDetails getUserByName(@PathVariable String name) {
         return userCrudOperationsService.loadUserByUsername(name);
     }
+    */
+
+    @PostMapping(value="/users", consumes=MediaType.APPLICATION_JSON_VALUE, produces=MediaType.APPLICATION_JSON_VALUE)
```

Comment:
to consumes produces w ogole nie potrzebne. Poza tym trzymajmy się konwencji @PostMapping("/users") nie dodajmy tego value